_docs/_latest/develop/test/integration-testing/run-test-suite.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Run The Test Suite Locally
+order: 045
+---
+
+# {{ page.title }}
+
+This page describes how to run the integration test suite for your Forseti
+contributions on your local dev environment. 
+
+## **Setting up development environment**
+
+- Create a new project in your organization.
+- Terraform uses an IAM Service Account to deploy and configure resources on 
+behalf of the user. You can create the Service account, and grant the necessary 
+roles to the Service Account manually or by running the helper script. 
+
+### **Creating Service Account by running the helper script**
+
+- In order to execute this script, you must have an Organization Administrator 
+Cloud Identity and Access Management (Cloud IAM) role so the script can assign
+the Forseti services account roles on the organization Cloud IAM policy.
+- Follow the instructions [here](https://forsetisecurity.org/docs/latest/setup/install/index.html#create-the-service-account-and-enable-required-apis)
+to create the Service Account, grant most of the required roles to the Service 
+Account, and enable the APIs by running the helper script. 
+- Please enable the following roles to the Service Account:
+
+On the organization:
+
+* roles/billing.user
+* roles/resourcemanager.folderViewer
+* roles/resourcemanager.organizationViewer
+* roles/resourcemanager.projectCreator
+
+On the project:
+
+* roles/cloudkms.admin
+
+Enable the following APIs on the Forseti project:
+
+* cloudkms.googleapis.com
```

Comment:
This API is only needed for the inventory perf test, and we are recommending people disable it. Might be good to put this info with the info about disabling the test, and then users won't have to do anything extra other than running the setup script.

Reply:
Done